proprietary/tokens/src/components/ams/heading.tokens.json

```diff
@@ -2,34 +2,32 @@
   "ams": {
     "heading": {
       "color": { "value": "{ams.color.text.default}" },
-      "font-family": { "value": "{ams.text.font-family}" },
-      "font-weight": { "value": "{ams.text.font-weight.bold}" },
+      "font-family": { "value": "{ams.headings.font-family}" },
+      "font-weight": { "value": "{ams.headings.font-weight}" },
       "inverse-color": { "value": "{ams.color.text.inverse}" },
-      "level": {
-        "1": {
-          "font-size": { "value": "{ams.text.level.1.font-size}" },
-          "line-height": { "value": "{ams.text.level.1.line-height}" }
-        },
-        "2": {
-          "font-size": { "value": "{ams.text.level.2.font-size}" },
-          "line-height": { "value": "{ams.text.level.2.line-height}" }
-        },
-        "3": {
-          "font-size": { "value": "{ams.text.level.3.font-size}" },
-          "line-height": { "value": "{ams.text.level.3.line-height}" }
-        },
-        "4": {
-          "font-size": { "value": "{ams.text.level.4.font-size}" },
-          "line-height": { "value": "{ams.text.level.4.line-height}" }
-        },
-        "5": {
-          "font-size": { "value": "{ams.text.level.5.font-size}" },
-          "line-height": { "value": "{ams.text.level.5.line-height}" }
-        },
-        "6": {
-          "font-size": { "value": "{ams.text.level.6.font-size}" },
-          "line-height": { "value": "{ams.text.level.6.line-height}" }
-        }
+      "level-1": {
```

Comment:
We should probably remove ‘level-’ here, just as we removed ‘color’ from contrast and inverse variations, and we don’t use ‘variant’ in the token name for primary button.